content/en/docs/reference/glossary/priority-class.md

```diff
@@ -0,0 +1,20 @@
+---
+title: PriorityClass
+id: priority-class
+date: 2024-03-19
+full_link: /docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass
+short_description: >
+  A mapping from a class name to the scheduling priority that a Pod should have.
+aka:
+tags:
+- core-object
+---
+A PriorityClass is a named class for the scheduling priority that should be assigned to a Pod
+in that class.
+
+<!-- more -->
```

Comment:
The content for this new glossary entry isn't rendering correctly in the [deploy preview (here)](https://deploy-preview-45608--kubernetes-io-main-staging.netlify.app/docs/reference/glossary/?core-object=true#term-priority-class). It seems Hugo is having trouble distinguishing between the short description and the summary content. Consider incorporating the suggested change below to address this, I think it's related to spacing in the comment.

```suggestion
<!--more-->
```